subsys/bluetooth/mesh/shell/shell_ctrl_cli.c

```diff
@@ -27,7 +27,7 @@ static int cmd_mode_get(const struct shell *shell, size_t argc, char *argv[])
 		return -ENODEV;
 	}
 
-	struct bt_mesh_light_ctrl_cli *cli = mod->user_data;
```

Comment:
This is not a unique light control client issue. I took a quick look over shell clients, looks, this is a general problem. Currently, almost all clients cannot be built. I'm not sure if it should be fixed in this PR, but obviously should.

Reply:
Created JIRA ticket for this. Let's do it in another PR.

Reply:
I will remove this part from this PR then.

Reply:
No, please, leave it as is. Otherwise, the client cannot be built